### ISSUE TITLE
onPress not ergonomic

### ISSUE THREAD

**Issue by jacobp100** [mpiannucci/react-native-context-menu-view#3]
The onPress event only gives back the title of what was pressed, which leads to code like this,

```js
onPress= (e) => {
  const {name} = e.nativeEvent;
  if (name === actions[0].title) {
    navigation.push(MemoryAdd, {item});
  } else if (name === actions[1].title) {
    onDelete();
  }
}
```

It would be more useful if we got an index

Or better if you look at [react-native-swipe-view](https://github.com/wix/react-native-swipe-view), each action has a callback

Is there one way you prefer, and do you want a hand implementing this?

**Comment by mpiannucci** [comment]
Gotcha. Yeah I had a specific reason for this when I was first writing it, but I can see the issue. The callback would work fine too, but I was trying to minimize the amount of event blocks being sent over the bridge. 

If you would like to take a stab at changing this how you think would be the most ergonomic that would be awesome.

**Comment by mpiannucci** [comment]
Closed with #4
